app/javascript/components/Statistic.js

```diff
@@ -1,9 +1,49 @@
 import React from "react";
+import { Mutation } from "react-apollo";
+import { DELETE_ACCOMPLISHMENT_MUTATION } from "../mutations/accomplishment_mutations";
 
 class Statistic extends React.Component {
+  state = {
+    deleted: false
+  };
+
+  hideStat = () => {
+    this.setState({ deleted: true });
+  };
+
   render() {
+    if (this.state.deleted) return <div>Poof gone!</div>;
     return (
       <div className="border border-grey-light rounded text-grey-darkest p-3 m-1 ml-0 max-w-sm">
+        {this.props.showRemove ? (
+          <Mutation
+            mutation={DELETE_ACCOMPLISHMENT_MUTATION}
+            variables={{
+              statisticId: this.props.id,
+              weekInReviewId: this.props.weekInReviewId
+            }}
+          >
```

Comment:
Do we also need to [update the cache](https://www.apollographql.com/docs/react/essentials/mutations.html#update) so that the accomplisment is removed off the screen?

Reply:
OK, so after reading up on that Apollo feature, and contrasting it against our case, I believe we would not need to update the cache.

Reason being, when we call the mutation, after we delete the `Accomplishment` db record, we `this.setState({ deleted: true });` for the `Statistic` component.  The immediate result of this is the statistic disappearing from the page, being replaced with "Poof gone!".

And then if you either stay on the same page, refresh the page, or navigate away and come back, they all result in _not_ seeing the deleted Statistic in the page anymore.

So I think we're all good.  Or no?